LaTeX/syntax_test_latex.tex

```diff
@@ -293,6 +293,14 @@ \subsubsection{name}
 }
 \end{lstlisting}
 
+\lstinline{var x = 15;}
```

Comment:
You should probably test here that the scope is closed (and the newline character does not have the meta scope anymoer).